package.json

```diff
@@ -8,11 +8,17 @@
   ],
   "scripts": {
     "prepublishOnly": "npm run lint && npm run test && npm run build",
-    "build": "babel src --out-dir dist",
+    "build": "tsc -p ./tsconfig.prod.json",
     "coverage": "nyc npm run test && nyc report --reporter=text-lcov > .nyc_output/lcov.info",
     "coveralls": "npm run coverage && coveralls <.nyc_output/lcov.info",
-    "lint": "standard",
-    "test": "babel-tape-runner ./test/*.js"
+    "format": "prettier --list-different **/*.{ts,json,md}",
+    "format:fix": "prettier --write **/*.{ts,json,md}",
+    "tslint": "tslint -p ./tsconfig.json -e node_modules/**/* -e **/node_modules/**/* -e dist/**/* **/*.ts",
+    "tslint:fix": "tslint --fix --format stylish -p ./tsconfig.json -e node_modules/**/* -e **/node_modules/**/* -e dist/**/* **/*.ts",
+    "tsc": "tsc --noEmit",
+    "lint": "npm run format && npm run tslint && npm run tsc",
+    "lint:fix": "npm run format:fix && npm run tslint:fix && npm run tsc",
+    "test": "ts-node node_modules/tape/bin/tape ./test/index.ts"
```

Comment:
We have now introduced these unified build scripts from ``ethereumjs-config``, see e.g. [in the account library](https://github.com/ethereumjs/ethereumjs-account/blob/master/package.json), please use these here.

Reply:
https://github.com/ethereumjs/ethereumjs-account/blob/10b46d6be6c87ff07d964e141f290eaa2c2c0f4a/package.json#L27 shows the same script? Which script should I use here?

Reply:
Ah, yes, the _other_ script entries. I'll update them!

Reply:
👍